modules/ivc_champva/app/services/ivc_champva/ves_data_formatter.rb

```diff
@@ -8,8 +8,10 @@ class VesDataFormatter
     VALID_RELATIONSHIPS_LOOKUP = RELATIONSHIPS.index_by(&:downcase).freeze
     VALID_GENDER_LOOKUP = { 'm' => 'MALE', 'male' => 'MALE', 'f' => 'FEMALE', 'female' => 'FEMALE' }.freeze
 
-    # Transform parsed form data from frontend format to VES format & validate
-    def self.format(parsed_form_data)
+    # Transform parsed form data from frontend format to a VES request & validate
+    # @param parsed_form_data [Hash] the parsed form data from the frontend
+    # @return [IvcChampva::VesRequest] the VES request object
+    def self.format_for_request(parsed_form_data)
```

Comment:
This was renamed from `format` because it was conflicting with the built in ruby method